### ISSUE TITLE
Text widget: Unable to save alt text in image properties for image added to widget

### ISSUE THREAD

**Issue by pauljacobson** [Automattic/wp-calypso#37214]
A user reported that they were unable to add `alt` text to an image's properties when adding an image to a Text widget in the Customizer.

The site is a WordPress.com site on a WordPress.com Premium plan, and is using the [Organization Theme](https://wordpress.com/theme/organization).

#### Steps to reproduce
1. The user created a Text widget, and added an image to it.
2. The user clicked on the pencil icon to edit the image's properties.
3. The user was able to add a link to the image, but adding `alt` text and saving the addition didn't actually add the `alt` text.
4. I was able to add the `alt` text by switching to the Text tab, and adding it in the HTML directly.

#### What I expected

Adding `alt` text in the visual editor, specifically the image properties window, should add `alt` text to the image properties when applied.

#### What happened instead

Adding the `alt` text, and saving the changes to the image properties didn't result in the `alt` text being applied to the image.

#### Browser / OS version

The user used Firefox 69.0 on Mac OS X 10.14.0 .
I'm using Firefox Developer Edition 71.0b4 on macOS 10.14.6

#### Screenshot / Video

![alt_text_Text_widget-16321090-hc](https://user-images.githubusercontent.com/2912202/67945683-30218b80-fbe8-11e9-8dcd-68f8d4876b57.png)

#### Context / Source

User report at 16321090-hc
